.github/workflows/regression.yml

```diff
@@ -128,7 +128,7 @@ jobs:
         uses: docker://ghcr.io/su2code/su2/test-su2:230225-2136
         with:
           # -t <Tutorials-branch> -c <Testcases-branch>
-          args: -b ${{github.ref}} -t develop -c develop -s ${{matrix.testscript}}
+          args: -b ${{github.ref}} -t develop -c feature_new_flamelet -s ${{matrix.testscript}}
```

Comment:
Just a note here to change this back before merging.
(Speaking from experience)

Reply:
I will change this back when merging and when the test cases branch is merged.